app/controllers/api/v1/sauces_controller.rb

```diff
@@ -1,5 +1,8 @@
 class Api::V1::SaucesController < ApiController
-  def show 
-    render json: Sauce.find(params[:id])
+  def show
+    sauce_show_obj = Sauce.find(params[:id])
+    
+    render json: sauce_show_obj
+
```

Comment:
delete line 6. I also suggest renaming your variable to something simpler, like `sauce`